site/styles/_plans.scss

```diff
@@ -83,3 +83,16 @@ s72-plan-label {
     display: inline !important;
   }
 }
+
+.s72-subscribe-trial-label {
+  display: none;
```

Comment:
I assume this is display:none by default but theres something that removes that to show it when its needed? just wondering if the other styles are insignificant or left in by accident. otherwise all good.